README.md

```diff
@@ -156,8 +156,8 @@ afterAll(() => {
 // hidden-message.js
 import React from 'react'
 
-// NOTE: React Testing Library works with React Hooks _and_ classes just as well
-// and your tests will be the same however you write your components.
+// NOTE: React Testing Library works well with React Hooks and classes.
+// Your tests will be the same whichever you write your components.
```

Comment:
I don't think `whichever` is grammatically correct here 

```suggestion
// Your tests will be the same however you write your components.
```

Reply:
Maybe `whichever` can be replaced with `regardless of how` 👀

Reply:
Thanks for the input, its updated. cc @nickmccurdy